config/migrations/2024/20241202142000-cleanup-worship-data/20241202142000-remove-betrokken-lokaal-bestuur-nieuwerkerken.sparql

```diff
@@ -0,0 +1,31 @@
+PREFIX ere: <http://data.lblod.info/vocabularies/erediensten/>
+PREFIX mu: <http://mu.semte.ch/vocabularies/core/>
+PREFIX org: <http://www.w3.org/ns/org#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFiX skos: <http://www.w3.org/2004/02/skos/core#>
+
+DELETE {
+  GRAPH ?g {
+    ?betrokkenLokaleBesturen a ere:BetrokkenLokaleBesturen ;
+      mu:uuid ?uuidBetrokkeLokaleBesturen ;
+      ere:financieringspercentage ?percentage ;
+      org:organization ?worshipAdminUnit ;
+      ere:typebetrokkenheid ?betrokkenheidsType .
+
+    ?bestuurseenheid ere:betrokkenBestuur ?betrokkeLokaleBesturen .
```

Comment:
**Issue:** There is an "n" missing in the object variable. It should be `?betrokkeNLokaleBesturen`. Otherwise the query deletes a bit to aggressively.